### PR TITLE
feat(client): add a customizable message to the dashboard page

### LINKED ISSUE #2434: Welcome page message is never displayed
The BIT shows some legal notices using this: https://github.com/SwissDataScienceCenter/renku/blob/0.24.1/helm-chart/renku/values.yaml#L609

But with the addition of the dashboard the welcome page is never shown.

We should either put this functionality back in or announce that it has been removed and change the values file in ui and renku repos accordingly.

### PR DESCRIPTION
**Pull request by leafty**
Fixes #2434.

renku-ci-ui-2592 dev renku ch_

/deploy #persist #cypress

### CODE REVIEW

File: client/src/features/dashboard/message/dashboardMessage.utils.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      ? "plain"
+      : rawStyle === "success"
+      ? "success"
+      : rawStyle === "info"
+      ? "info"
+      : rawStyle === "warning"
+      ? "warning"
+      : rawStyle === "danger"
+      ? "danger"
+      : null;
```
[the remark this answers is not in the document]
Comment (leafty) [reply client/src/features/dashboard/message/dashboardMessage.utils.ts:71]: Good point :+1:, using "plain" whenever we cannot parse the `style` option as any allowed value is a better experience for the instance admin.

**Comment by leafty** [comment]
> This is implemented nicely! My only comments are a correction in the wording of a comment and the suggestion to default the style.

@ciyer Thanks for the suggestions, they have been added. I also included new e2e tests for this feature.

**Review by ciyer — DISMISSED** [review]
The JupyterNotebookForPrivateProjectSpec Selenium test is failing, but that is expected and should automatically resolve itself once renku-python is tagged, so we can merge this.

File: helm-chart/renku-ui/values.yaml [hunk 2]
```diff
@@ -97,11 +97,29 @@ client:
...
+    # Style of the message card. Allowed values are:
+    # - "plain" -> simple white background card, no icon
+    # - "success" -> green background, green checkmark icon
+    # - "info" -> blue background, "i" in circle icon
+    # - "warning" -> yellow background, triangle exclamation mark icon
+    # - "danger" -> red background, triangle exclamation mark icon
+    style: info
+    # When this is set to true, users can dismiss the message and hide
+    # it form the dashboard page until the tab is refreshed.
+    dismissable: true
```
Comment (lorenzo-cavazzi) [inline helm-chart/renku-ui/values.yaml:122]: This seems to be a typo, it should be `dismissible`.
I was initially confused by the tests working properly and the CI deployment showing no dismiss button despite it being set in the parameters :joy: 
image

**Review by lorenzo-cavazzi — CHANGES_REQUESTED** [review]
Good implementation :raised_hands: 
I spotted a typo that led to an unexpected default setting (see inline).

An additional comment: once we expand the text, the arrow direction changes but the text still says "Read more...". Should we change the text too?

**Review by lorenzo-cavazzi — DISMISSED** [review]
Lgtm! :rocket:

**Comment by lorenzo-cavazzi** [comment]
Oh no, I accidentally pushed a commit to the wrong PR :facepalm:  Sorry for that! I'll remove it immediately

**Pull request merged**